### ISSUE TITLE
Doesn't work.

### ISSUE THREAD

**Issue by NoxiWave** [crpmax/minecraft-bot-flood#7]
Is it even working for this days? I tried it, it "created" bots. And after some time program just ends without any bots on the Server.

**Comment by crpmax** [comment]
Hello, please read README

> This script is used for stress testing Minecraft servers, it can connect a lot of bots very fast. These bots will not respond to any server packets, so they will time out after 30 seconds by default.

**Comment by NoxiWave** [comment]
I saw, but I wanted try this project, if it's not working, atleast include it.

**Comment by crpmax** [comment]
What MC version / protocol version are you using?

**Comment by NoxiWave** [comment]
> What MC version / protocol version are you using?

I tried 1.19.4 and 1.20.4

**Comment by NoxiWave** [comment]
Could you fix it to work? Or make something smilliar in Java?

**Comment by crpmax** [comment]
Yes, you were right, it is not working on latest versions. I am working on a fix, I fixed it on MC 1.19 to 1.20.1 but on 1.20.2 and up there is some problem.

**Comment by NoxiWave** [comment]
It is possible to fix the upper versions?

**Comment by crpmax** [comment]
Download the new version and try it now.

**Comment by NoxiWave** [comment]
I will try

**Comment by NoxiWave** [comment]
It's seem working, also do you plan adding proxy support?

**Comment by NoxiWave** [comment]
For me also seem only 1 bot join not more

**Comment by crpmax** [comment]
Proxy support is possible, but because it is written in PHP, it would slow down all connections. I do not plan to add proxy support.

Do you have set connection throttle to -1 on the server?

**Comment by NoxiWave** [comment]
Yea now I figured it out. By the way what about rewriting this project into Java and then adding proxy support? It would be good

**Comment by skymah** [comment]
What's the protoversion of 1.20.1 I can't seem to find the number online
